RocketApp/Infrastructure/Sources/NetworkClientExtensions/DependencyValues+.swift

```diff
@@ -0,0 +1,10 @@
+import Dependencies
+import Foundation
+import Networking
+
+public extension DependencyValues {
```

Comment:
Tenhle soubor by se mel jmenovat `DependencyValues+networkClientType`, tohle je moc obecny, to samy v Rocket clientovi...

Reply:
Dependency values a keys slouceny do Client+dependencyManaging.